node/tests/TestUtilities.ts

```diff
@@ -1051,7 +1062,7 @@ export async function transactionTest(
     baseTransaction.bitpos(key17, 1);
     responseData.push(["bitpos(key17, 1)", 1]);
 
-    if (gte("6.0.0", version)) {
+    if (gte(version, "6.0.0")) {
```

Comment:
we don't support anything less than 6...

Reply:
At least we should properly check the version, otherwise it is never tested